src/ui/js/Game.js

```diff
@@ -2791,11 +2797,55 @@ Game.createGameMatDieDiv = function(die, player, dieStatus, isClickable) {
     dieNumberSpanOpts.html = '&nbsp;' + die.value + '&nbsp;';
   }
 
+  divOpts.style = 'background-image: ' + Game.backgroundImagePath(
+    die.sides,
+    dieStatus,
+    isClickable
+  );
+
   var dieDiv = $('<div>', divOpts);
   dieDiv.append($('<span>', dieNumberSpanOpts));
+
   return dieDiv;
 };
 
+/*
+ * Calculate the background image path
+ *
+ * @param  int     sides          Number of sides of the die
+ * @param  string  dieStatus      Status of the die ('active' or 'captured')
+ * @param  boolean isClickable    Is the die clickable?
+ * @return string
+ */
+Game.backgroundImagePath = function(sides, dieStatus, isClickable) {
+  var imageType;
+
+  var sidesRoundedArray = [2, 4, 6, 8, 10, 12, 20, 30];
+  var sidesRoundedUp = Math.max.apply(null, sidesRoundedArray);
```

Comment:
IMO, there'd be nothing wrong with `var sidesRoundedUp = sidesRoundedArray[sidesRoundedArray.length - 1];` if you didn't want to use Math functions, and it'd be slightly more clear (just because the null business is a little confusing).  Your code below assumes the array is sorted, so there's no point stressing about the case in which it isn't.